<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCenterWindow</class>
 <widget class="QWidget" name="EditCenterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/edit_icon.png</normaloff>:/icons/edit_icon.png</iconset>
  </property>
  <widget class="QGroupBox" name="loginGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Введіть нові дані про центр</string>
   </property>
   <widget class="QLabel" name="streetLabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Вулиця</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="streetLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="houseLabel">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер будинку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="houseLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="generalPlacesLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="generalPlacesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Загальна кіл-сть місць</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Редагувати центр</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
